charts/refractr/templates/hpa.yaml

```diff
@@ -12,5 +12,21 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: {{ .Values.deployment.name }}
-  targetCPUUtilizationPercentage: {{ .Values.hpa.target_cpu_utilization }}
+  metrics:
```

Comment:
These change should be not needed if you want to only scale based on CPU, as the previous configuration should work.
Anyway, this feature (custom metrics) is not available in `autoscaling/v1`. If you want to add it, you have to promote it to `autoscaling/v2beta2`.
More on that topic [on the API object definition](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#api-object)